src/components/FAQ/component.tsx

```diff
@@ -29,7 +29,7 @@ const FAQEntryComponent: FunctionComponent<{
   <div>
     <h5>{question}</h5>
     <div className="pb-5">
-      <ReactMarkdown source={answer} />
+      <ReactMarkdown source={answer} containerTagName="p" />
```

Comment:
I wrapped the markdown content in a `p` tag since the styling for `a`s is only then applied. See:
https://github.com/hardforkio/homepage/blob/afd6fe53bdc2e507af98d7e2ce2ee1d1f5066ce5/src/scss/main.scss#L4

Another option would be to apply the styles for all `a`s and add a `border-0` to links where the dotted bottom border is not desired.